src/pages/Swap/index.tsx

```diff
@@ -327,15 +328,17 @@ export function Swap({
 
   const handleTypeInput = useCallback(
     (value: string) => {
+      maybeLogFirstSwapAction(trace)
       onUserInput(Field.INPUT, value)
```

Comment:
<nit> maybe best to switch the order, so there's no chance that `onUserInput` is delayed by logging